cloud-etl/README.md

```diff
@@ -0,0 +1,121 @@
+
+This demo showcases an entire end-to-end cloud ETL deployment, built for 100% cloud services:
+
+* Kinesis source connector: reads from a Kinesis stream and writes the data to a Kafka topic in Confluent Cloud
+* ksqlDB: streaming SQL engine that enables real-time data processing against Kafka
+* GCS or S3 sink connector: pushes data from Kafka topics to cloud storage
+
+| Component                  | Consumes From              | Produces To             |
+|----------------------------|----------------------------|-------------------------|
+| Kinesis source connector   | Kinesis stream `demo-logs` | Kafka topic `eventLogs` |
+| KSQL                       | `eventLogs`                | KSQL streams and tables ([ksql.commands](ksql.commands)) |
+| GCS/S3/Blob sink connector | KSQL tables `COUNT_PER_SOURCE`, `SUM_PER_SOURCE` | GCS/S3/Blob         |
+
+# Warning
+
+This demo uses real cloud resources, including that of Confluent Cloud, Amazon Kinesis, and one of the cloud storage providers.
+To avoid unexpected charges, carefully evaluate the cost of resources before launching the demo and ensure all resources are destroyed after you are done running it.
+
+# Prerequisites
+
+## Cloud services
+
+* [Confluent Cloud cluster](https://confluent.cloud?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl): for development only. Do not use a production cluster.
+* [Confluent Cloud KSQL](https://docs.confluent.io/current/quickstart/cloud-quickstart/ksql.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl) provisioned in your Confluent Cloud
+* AWS or GCP or Azure access
+
+## Local install
+
+* [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud) v0.234.0 or later
+* AWS S3: `aws` CLI, properly initialized with your credentials
+* GCP GCS: `gsutils` CLI, properly initialized with your credentials
+* Azure Blob: `az` CLI, properly initialized with your credentials
+* `jq`
+* `curl`
+
+# Run the Demo
+
+## Setup
+
+1. Initialize a properties file at `$HOME/.ccloud/config` with the following configuration parameters for your Confluent Cloud cluster:
+
+```shell
+$ cat $HOME/.ccloud/config
+bootstrap.servers=<BROKER ENDPOINT>
+ssl.endpoint.identification.algorithm=https
+security.protocol=SASL_SSL
+sasl.mechanism=PLAIN
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
+schema.registry.url=https://<SR ENDPOINT>
+schema.registry.basic.auth.user.info=<SR API KEY>:<SR API SECRET>
+basic.auth.credentials.source=USER_INFO
+ksql.endpoint=https://<KSQL ENDPOINT>
+ksql.basic.auth.user.info=<KSQL API KEY>:<SR KSQL SECRET>
+```
+
+To get the right values for the endpoints in the file above, find them either via the Confluent Cloud UI or Confluent Cloud CLI commands:
```

Comment:
If the customer has multiple Kafka clusters, they should make sure to specify the one with the associated KSQL app.

Reply:
@bechols good point, thanks.